File: product/src/main/scala/de/leanovate/dose/product/model/ProductOption.scala
package de.leanovate.dose.product.model

import spray.json.DefaultJsonProtocol

case class ProductOption(
                          name: String,
                          description: Option[String],
                          priceInCent: Int
                          )

object ProductOptionProtocol extends DefaultJsonProtocol {
  implicit val productOptionFormat = jsonFormat3(ProductOption)
}

File: billing/src/main/java/de/leanovate/dose/billing/model/ProductOption.java
package de.leanovate.dose.billing.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class ProductOption {
    @JsonProperty
    public String name;

    @JsonProperty
    public String description;

    @JsonProperty
    public int priceInCent;

    @Override
    public String toString() {

        return "ProductOption{" +
                "name='" + name + '\'' +
                ", description='" + description + '\'' +
                ", priceInCent=" + priceInCent +
                '}';
    }
}
